util/datamodel.py

```diff
@@ -87,6 +87,20 @@ def add_location_data(ds, lat, lon):
     ds['lon'] = ('obs', lon_array, {'axis': 'X', 'units': 'degrees_east', 'standard_name': 'longitude'})
 
 
+def find_depth_variable(variable_list):
+    """
+    Find the depth variable in the variable_list
+    :param variable_list: the list of variable names
+    :return: the depth variable, or None if the variable_list contains no known depth variable
+    """
+    depth_variable=None
```

Comment:
The only reason `depth_variable` would not get set is if `app.config['NETCDF_DEPTH_VARIABLES']` returns `None`, in which case an untrapped exception will occur. If you want to protect against that, use `get`:

```python
for variable in app.config.get('NETCDF_DEPTH_VARIABLES', []):
    if variable in variable_list:
        depth_variable = variable
            break```

Reply:
will do

Reply:
done